scripts/variantstore/docs/aou/AOU_DELIVERABLES.md

```diff
@@ -52,27 +50,32 @@
    - This workflow does not use the Terra Data Entity Model to run, so be sure to select the `Run workflow with inputs defined by file paths` workflow submission option.
-8. `GvsExtractCallset` workflow
-   - This workflow extracts the data in BigQuery and transforms it into a sharded joint called VCF incorporating the VQSR filter set data.  We will probably not run this on callsets of more than 100K samples.
-   - It also needs to be run twice, once with `control_samples` set to "true", and with the `filter_set_name` and `extract_table_prefix` from step 5 & 6.  Include a valid (and secure) "output_gcs_dir" parameter, which is where the VCF, interval list, manifest, and sample name list files will go.
-   - This workflow does not use the Terra Data Entity Model to run, so be sure to select the `Run workflow with inputs defined by file paths` workflow submission option.
-9. **TBD VDS Prepare WDL/notebook/??**
-10. **TBD VDS Extract WDL/notebook/??**
-11. Run the notebook to create callset stats (**TODO**, see [VS-388](https://broadworkbench.atlassian.net/browse/VS-388)), for which you need
-     - "BigQuery Data Viewer" role for your @pmi-ops proxy group on the `spec-ops-aou:gvs_public_reference_data.gnomad_v3_sites` table
-     - the Google project ID you used for all the GVS WDLs (`project_id` input)
-     - the name of the BigQuery dataset you used for all the GVS WDLs (`dataset_name` input)
-     - the `extract_table_prefix` input from `GvsExtractCallset` step
-     - the `filter_set_name` input from `GvsCreateFilterSet` step
-12. [GvsCalculatePrecisionAndSensitivity](../../tieout/AoU_PRECISION_SENSITIVITY.md#generating-callset-precision-and-sensitivity-values) workflow, for which you need
-    - "Storage Object View" access granted for your @pmi-ops proxy group on the `gs://broad-dsp-spec-ops/gvs/truth` directory
-13. `GvsCallsetCost`
-    - The cost from this callset, which represents the total BigQuery cost (which is not represented in the Terra UI total workflow cost) from the GVS pipeline workflows, is used to calculate the cost of the callset as a whole and by sample.
-
+8. Run the VDS Extract using a notebook terminal and the python script called `hail_gvs_import.py` from the GenerateHailScripts task. It will look something like: `gs://fc-<workspace-id>/submissions/<submission id>/GvsExtractAvroFilesForHail/<workflow id>/call-GenerateHailScripts/hail_gvs_import.py`
+    - This step creates a VDS based on the Avro files 
+    - Notebook provisioning suggestions and other necessary set up details (e.g. the Hail wheel to use) can be found in the file: [AoU Delta VDS Cluster Configuration.md](vds/cluster/AoU%20Delta%20VDS%20Cluster%20Configuration.md)
+    - We suggest gsutil cp-ing `hail_gvs_import.py` to the notebook and then invoking it directly in the terminal
+    - inputs to the python script are:
+      1. `--avro-path`: the directory path at which exported GVS Avro files are found in GCP
```

Comment:
the version of the script generated at `GenerateHailScripts/hail_gvs_import.py` should have set these to the correct defaults

Reply:
this needs to be investigated---it doesn't seem to be the case currently?